planner-agent/src/workflows/todocreator/__main__.py

```
@@ -11,6 +11,8 @@
 from src.server.logging_setup import setup_remote_logging
 from src.workflows.todocreator.utils import SwarmBountyType
 
+from src.workflows.todocreator.utils import SwarmBountyType
```

Ruff (0.11.9)

Redefinition of unused `SwarmBountyType` from line 12

Remove definition: `SwarmBountyType`

(F811)